db/migrations/20200407131156_create_median_drop.sql

```diff
@@ -0,0 +1,25 @@
+-- +goose Up
+
+CREATE TABLE maker.median_drop
+(
+    id         SERIAL PRIMARY KEY,
+    log_id     BIGINT  NOT NULL REFERENCES public.event_logs (id) ON DELETE CASCADE,
+    header_id  INTEGER NOT NULL REFERENCES public.headers (id) ON DELETE CASCADE,
+    address_id INTEGER NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    msg_sender INTEGER NOT NULL REFERENCES public.addresses (id) ON DELETE CASCADE,
+    a_length   INTEGER NOT NULL,
+    a          TEXT ARRAY NOT NULL,
+    UNIQUE (header_id, log_id)
+);
+
+CREATE INDEX median_drop_log_index
+    ON maker.median_drop (log_id);
+CREATE INDEX median_drop_header_index
+    ON maker.median_drop (header_id);
+CREATE INDEX median_drop_address_index
+    ON maker.median_drop (address_id);
+CREATE INDEX median_drop_msg_sender_index
+    ON maker.median_drop (msg_sender);
+-- +goose Down
+
+DROP TABLE maker.median_drop;
```

Comment:
sorry to be a pain, but I think it'd be good to include the changes to the db/schema in this PR if possible. Happy to pull down the branch and add it myself if you're running a version other than 11.6 and that'd be helpful